.travis.yml

```diff
@@ -9,6 +9,8 @@ jdk:
   - openjdk7
   - openjdk6
 
+sudo: false
```

Comment:
I have confirmed the all build jobs success at my repository.
https://travis-ci.org/zaneli/slf4s/builds/102801324
But this PR failed.
https://travis-ci.org/mattroberts297/slf4s/builds/102801800

I guess this this difference occur because of Travis CI infrastructure types.
https://docs.travis-ci.com/user/workers/container-based-infrastructure/
> For repos we recognize before 2015-01-01, linux builds are sent to our standard infrastructure.
> For repos we recognize on or after 2015-01-01, linux builds are sent to our container-based infrastructure.

So I added `sudo: false` explicitly.